spec/tasks/backfill_working_patterns_details_spec.rb

```diff
@@ -0,0 +1,120 @@
+require "rails_helper"
+
+RSpec.describe "vacancies:backfill_working_patterns_details" do
+  let(:migration_date) { Date.new(2022, 8, 25) }
+  let(:full_time_details) { "Full time details" }
+  let(:part_time_details) { "Part time details" }
+  let(:working_patterns_details) { nil }
+
+  let!(:vacancy) do
+    create(:vacancy, created_at:, working_patterns:, full_time_details:, part_time_details:, working_patterns_details:)
+  end
+
+  RSpec.shared_examples "removes legacy working patterns details" do
```

Comment:
Doesn't look like this shared example is used anywhere

Reply:
Copy&Paste :facepalm: . Thanks for pointing!